### DISCUSSION TITLE
Link to diff or commit history on digest change

### DISCUSSION THREAD

**Discussion by JP-Ellis** [renovatebot/renovate#31403] [Suggest an Idea]
## Summary

Digest changes are the least informative, and it would be good to have a link review the upstream changes. Preferably a link to the commit history, but falling back to a straight diff.

## Background

In the context of GitHub Actions, though I suspect this may apply to other cases too, I frequently have updates of the form:

> This PR contains the following updates:
> 
> | Package | Type | Update | Change |
> |---|---|---|---|
> | [peter-evans/create-pull-request](https://redirect.github.com/peter-evans/create-pull-request) | action | digest | `8867c4a` -> `d121e62` |

This is because upstream has re-tagged the version.

Understandably, Renovate does not really have a changelog to link to, as this was an 'upgrade' from `vX` to `vX`.

As a fallback to having a changelog, it would be good to link to either a diff or a commit history:

- [diff](https://github.com/peter-evans/create-pull-request/compare/8867c4a..d121e62)
  ```
  https://github.com/peter-evans/create-pull-request/compare/8867c4a..d121e62
  ``` 
- [history](https://github.com/peter-evans/create-pull-request/compare/8867c4a...d121e62)
  ```
  https://github.com/peter-evans/create-pull-request/compare/8867c4a...d121e62
  ```

I would _not_ suggest including the commits within the PR comment itself, because diffs (and occasionally commit histories) could become extremely long; however, a link would suffice.

## Related Discussions

I tried search for this, and I could not find much. The only relevant link so far is:

- https://github.com/renovatebot/renovate/discussions/24903

**Comment by rarkins** [comment]
Columns in Renovate are fully customizable, so you might be able to already add logic to achieve this. But this can also be useful default behavior too.

**Reply by JP-Ellis** [reply]
I'm not sure what the action is here. Is this a default behaviour which should be changed in renovate? Or are you saying this is something I need to configure? Or are you asking I make a PR?

**Reply by rarkins** [reply]
You should try solving this through configuration using prBodyDefinitions

**Comment by jamietanna** [comment]
Added as part of #39650
